pkg/standalone/client.go

```diff
@@ -0,0 +1,32 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package standalone
+
+type DaprProcess interface {
+	List() ([]ListOutput, error)
+}
+
+type daprProcess struct {
+}
+
+type Client interface {
+	DaprProcess
+	InvokeGet(appID, method string) (string, error)
+	InvokePost(appID, method, payload string) (string, error)
+	Publish(topic, payload, pubsubName string) error
+}
+
+type Standalone struct {
+	process DaprProcess
+}
+
+func NewStandaloneClient() Client {
+	return &Standalone{process: &daprProcess{}}
+}
```

Comment:
Checkout "Avoid stutter" here https://blog.golang.org/package-names (tl;dr in many Go packages the naming convention is `http.Server`, not `http.HttpServer` similarly `httptest.Server`, not `httptest.HttpTestServer`).  So in this case, calling `standalone.NewClient()` would be enough to have no ambiguity.

```suggestion
func NewClient() Client {
	return &Standalone{process: &daprProcess{}}
}
```

Reply:
Also here, the review comment was resolved without a change or an objection.